src/styles/index.scss

```diff
@@ -1,3 +1,222 @@
+@import './utils/variables';
+@import './utils/mixins';
+
+html,
+body,
+main,
+div,
+header,
+ul,
+li,
+a,
+h1,
+p,
+img {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  @include font($size: 10px, $weight: 400, $color: $main-accent-color);
+
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+button {
+  background-color: transparent;
+  color: inherit;
+  border-width: 0;
+  padding: 0;
+  cursor: pointer;
+}
+
+h1,
+p {
+  font-size: inherit;
+  font-weight: inherit;
+}
+
+ul,
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  @include flex(row, space-between, center);
+
+  max-width: 1200px;
+  margin: 0 auto;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  &__logo {
+    img {
+      @include set-size(40px, 40px);
```

Comment:
Ensure that the mixin `set-size` is defined in your mixins file. If it's not defined, this will cause an error during SCSS compilation.